user/languages/go.md

```diff
@@ -50,19 +50,28 @@ handled by [gimme](https://github.com/travis-ci/gimme).
 language: go
 
 go:
-  - "1.x"
-  - "1.8"
-  - "1.10.x"
-  - master
+- 1.x
+- "1.10"
+- 1.11.x
+- master
 ```
 {: data-file=".travis.yml"}
 
 
+## Go Modules
+
+If Go 1.11+ is specified and either `GO111MODULE=on` is set or a `go.mod` file
+is present, then it is assumed that [Go
```

Comment:
`go.mod` should exist at the repository root, correct?

Reply:
@BanzaiMan yes, but I didn't feel that explaining the rules here was appropriate given that they aren't _our_ rules